lib/ansible/runner.rb

```diff
@@ -218,7 +218,7 @@ def run_via_cli(hosts, credentials, env_vars, extra_vars, tags: nil, ansible_run
           fetch_galaxy_roles(playbook_or_role_args)
 
           result = wait_for(pid_file(base_dir)) do
-            AwesomeSpawn.run("ansible-runner", :env => env_vars_hash, :params => params)
+            AwesomeSpawn.run!("ansible-runner", :env => env_vars_hash, :params => params)
```

Comment:
If i recall, this didn't use `!` on purpose because of some weird cases where it returns 1 as a "good" return value.

Reply:
Hm okay, let me just update the timeout to match jansa and we can see if there's a better way to do this in a follow-up